dataactcore/logging.py

```diff
@@ -130,6 +130,7 @@ def format(self, record):
             record.asctime = self.formatTime(record, self.datefmt)
         s = self.formatMessage(record)
         if record.exc_info:
+            print(record.exc_info)
```

Comment:
Oops! That was from when CM and I were trying to figure out why I was getting a unit test error and he wasn't (3.4 vs 3.5 issue)